src/reducers/view/view.js

```diff
@@ -196,15 +196,21 @@ export function collapseRow(state, action) {
 export function expandRow(state, action) {
   const { expandedRows } = state;
   const item = action.value;
+  let newState;
 
   if (!expandedRows.includes(item)) {
-    expandedRows.push(item);
+    newState = {
+      ...state,
+      expandedRows: [...expandedRows, item],
+    };
+  } else {
+    newState = {
+      ...state,
+      expandedRows,
+    };
   }
```

Comment:
if we're not modifying anything, should we just return `state` ?

Reply:
Yes good catch! This has been updated accordingly.